tensorboard/components/tf_card_heading/tf-card-heading.html

```diff
@@ -62,7 +62,10 @@
         <template is="dom-if" if="[[_tagLabel]]">
           <div class="heading-row">
             <div class="heading-label">
-              tag: <span itemprop="tag">[[_tagLabel]]</span>
+              tag: <div
+                     itemprop="tag"
```

Comment:
Why not run too? Possibly also do name as well.

Reply:
Run was just an oversight because it's not in the scalars plugin card headers.  I'll redo some test data with one of the plugins that does show runs in the headers and add this behavior to run as well.

Re: name, that didn't seem as useful since custom display names can easily have spaces in them (and it'd be a little weird to break mid-word when there are space breaks available, IMO - unlike a book we're not trying to justify text to fill a column).  We could potentially insert `<wbr>` only for display names that don't already have whitespace, but overall this seems less important than the run or the tag because 1) human-choosen names will tend to be short and 2) automatic names are based only on the node name without any enclosing namespaces, so they'll also tend to be much shorter than the tag name (or run name).